src/sagemaker/clarify.py

```diff
@@ -1068,50 +1152,65 @@ def run_explainability(
         kms_key=None,
         experiment_config=None,
     ):
-        """Runs a ProcessingJob computing for each example in the input the feature importance.
-
-        Currently, only SHAP is supported as explainability method.
+        """Runs a :class:`~sagemaker.processing.ProcessingJob` computing feature attributions.
 
         Spins up a model endpoint.
-        For each input example in the 's3_data_input_path' the SHAP algorithm determines
-        feature importance, by creating 'num_samples' copies of the example with a subset
-        of features replaced with values from the 'baseline'.
-        Model inference is run to see how the prediction changes with the replaced features.
-        If the model output returns multiple scores importance is computed for each of them.
-        Across examples, feature importance is aggregated using 'agg_method'.
+
```

Comment:
SHAP and PDP could be requested together as well. Maybe try to reflect that as well here.

Reply:
Thank you, added this to my upcoming PR.